static/options.js

```diff
@@ -23,4 +23,18 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 "use strict";
+
+var configElement = document.getElementById('config');
```

Comment:
can we not "just" in the HTML file have a `<script>window.configRoot=...` instead of relying on putting JSON text inside strings inside DOM elements? Or am I missing something?

Reply:
Got it!! Perfect.